Dockerfile.deps

```diff
@@ -8,7 +8,10 @@ RUN apk add --no-cache \
       make \
       bash \
       gcc \
-      musl-dev
+      g++ \
+      libsass-dev \
+      musl-dev \
+      && true
```

Comment:
This got remove, it's nice to have it as it allows package names to be sorted easily.

Reply:
I intentionally removed it. It makes the installation not to fail if something goes wrong. It happened to me and I spent quite some time debugging the reason.

Reply:
> It makes the installation to fail silently if something goes wrong 

No, because it's `&& true`, not `|| true`.